src/lib/ActionPrototype.js

```diff
@@ -1,29 +1,57 @@
-var fetcher = require('../lib/jsonFetcher');
-var AppDispatcher = require('../lib/ep-dispatcher');
-var KeyMirror = require('keymirror');
+import fetcher from '../lib/jsonFetcher'
```

Comment:
One of the first bits of the entire codebase, we had no standards back then. Just an annoying legacy. Something to get caught up at somepoint.